Tools/WebKitTestRunner/CMakeLists.txt

```diff
@@ -19,15 +19,19 @@ set(WebKitTestRunner_SOURCES
     WorkQueueManager.cpp
 )
 
-set(WebKitTestRunner_LIBRARIES
-    TestRunnerShared
-)
+set(WebKitTestRunner_LIBRARIES TestRunnerShared)
+
 set(WebKitTestRunner_FRAMEWORKS
     JavaScriptCore
+    PAL
+    WTF
     WebCore
     WebCoreTestSupport
     WebKit
 )
+if (NOT USE_SYSTEM_MALLOC)
+    list(APPEND JavaScriptCore_FRAMEWORKS bmalloc)
```

Comment:
Is this `JavaScriptCore_FRAMEWORKS` a typo of `WebKitTestRunner_FRAMEWORKS`?

Reply:
Copy-pasta error. Good catch! Fixed now